winui/Rating/Precision.md

```diff
@@ -61,7 +61,7 @@ rating.ItemsCount = 5;
 
 ## Exact
 
-Rating value can be provided with exact precision.
+Rating value can be provided with exact value.
```

Comment:
The rating value can be provided with the exact value.

Reply:
Changed the content as mentioned above.